offline-tools/copyrighter/pkg/cmd/galasacopyrighter.go

```diff
@@ -28,6 +28,8 @@ const (
 	COMMENT_START_JAVA    = "/*"
 	COMMENT_CONTINUE_JAVA = " *"
 	COMMENT_END_JAVA      = " */"
+	COMMENT_DOUBLE_STROKE = "//"
```

Comment:
This isn't being used, so needs to be removed.

Reply:
removed